modules/inherit.jsm

```diff
@@ -21,23 +21,23 @@ function toPropertyDescriptors(aProperties) {
 	return descriptors;
 }
 
-function inherit(aParent, aExtraProperties) {
+function inherit(apparent, aExtraProperties) {
 	var global;
```

Comment:
This is not a typo. "aParent" means "argument: parent", a coding style of Mozilla.